### ISSUE TITLE
Implement filtering of tables to select for transfer

### ISSUE THREAD

**Issue by perlun** [perlun/pg2couch#1]

**Comment by krainboltgreene** [comment]
I would not limit this to tables. Arbitrary queries, materialized views, and views are *ideal* targets for one-way replication.

**Comment by perlun** [comment]
@krainboltgreene Makes sense! Feel free to submit a PR, I'm unfortunately not working on this active any more. (it was an experiment that never made its way out to production)
